batch/batch/aioclient.py

```diff
@@ -318,16 +322,29 @@ def create_job(self, image, command=None, args=None, env=None, ports=None,
             raise ValueError("cannot submit an already submitted batch")
         self._submitted = True
 
-        doc = {}
-        if self.attributes:
-            doc['attributes'] = self.attributes
-        if self.callback:
-            doc['callback'] = self.callback
-
-        if len(self._job_docs) != 0:
-            doc['jobs'] = self._job_docs
+        with aiohttp.MultipartWriter('batch') as mpwriter:
+            batch_doc = {'n_jobs': len(self._job_docs)}
+            if self.attributes:
+                batch_doc['attributes'] = self.attributes
+            if self.callback:
+                batch_doc['callback'] = self.callback
+            mpwriter.append_json(batch_doc)
+
+            docs = []
+            n = 0
+            for jdoc in self._job_docs:
+                n += 1
+                docs.append(jdoc)
+                if n == job_array_size:
```

Comment:
This doesn't seem like it guarantees that we stay under request limit, but I guess now that we don't have a limit specified for client_max_body_size, this is fine?

Reply:
Konrad was still seeing errors this morning even after that change. I figured we'd successfully sent batches of jobs with 81 jobs through ci, so 50 seemed like a safe number.